alchemy_news/appbuilder/alchemy-news-trend-endpoint.rb

```diff
@@ -0,0 +1,29 @@
+news_url = "http://access.alchemyapi.com/calls/data/GetNews"
+
+news_params = {
+  :apikey => 'YOUR_API_KEY_HERE',
+  :outputMode => 'json',
+  :count => 10
```

Comment:
What does the 'count' parameter do here?  I honestly can't find an answer in the docs...